crd/multitenancy/api/v1alpha1/multitenantpodnetworkconfig.go

```diff
@@ -85,6 +85,8 @@ type MultitenantPodNetworkConfigStatus struct {
 	// InterfaceInfos describes all of the network container goal state for this Pod
 	// +kubebuilder:validation:Optional
 	InterfaceInfos []InterfaceInfo `json:"interfaceInfos,omitempty"`
+	// DefaultDenyAcl bool indicates whether default deny policy will be present on the pods upon pod creation
```

Comment:
nit: comment should start with field name matching exactly
```suggestion
	// DefaultDenyACL bool indicates whether default deny policy will be present on the pods upon pod creation
```

Reply:
changed it!